File: solutions/3533-snake-in-matrix/solution.java
class Solution {
    public int finalPositionOfSnake(int n, List<String> commands) {
        int j=0;
        for(int i=0;i<commands.size();i++)
        {
            if(commands.get(i).equals("RIGHT"))
                j++;
            if(commands.get(i).equals("LEFT"))
                j--;
            if(commands.get(i).equals("UP"))
                j-=n;
            if(commands.get(i).equals("DOWN"))
                j+=n;
        }
        return j;
    }
}

File: solutions/3533-snake-in-matrix/solution.py
class Solution(object):
    def finalPositionOfSnake(self, n, commands):
        """
        :type n: int
        :type commands: List[str]
        :rtype: int
        """
        k=0
        for i in commands:
            if i=="RIGHT":
                k+=1
            elif i=="LEFT":
                k-=1
            elif i=="UP":
                k-=n
            else:
                k+=n
        return k
